mrtt-ui/src/views/LandscapeForm.js

```diff
@@ -1,14 +1,197 @@
+  organizations: yup.array()
+})
+
+const formDefaultValues = { landscape_name: '', selectedOrganizations: [] }
 
 const LandscapeForm = ({ isNewLandscape }) => {
+  const [doesLandscapeExist, setDoesLandscapeExist] = useState(true)
+  const [isLoading, setIsLoading] = useState(false)
+  const [isSubmitError, setIsSubmitError] = useState(false)
+  const [isSubmitting, setIsSubmitting] = useState(false)
+  const [organizationOptions, setOrganizationOptions] = useState([])
   const { landscapeId } = useParams()
-  return (
+  const landscapesUrl = `${process.env.REACT_APP_API_URL}/landscapes`
+  const landscapeUrl = `${landscapesUrl}/${landscapeId}`
+  const navigate = useNavigate()
+  const organizationsUrl = `${process.env.REACT_APP_API_URL}/organizations`
+
+  const {
+    control: formControl,
+    handleSubmit: validateInputs,
+    formState: { errors },
+    reset: resetForm
+  } = useForm({ resolver: yupResolver(validationSchema), defaultValues: formDefaultValues })
+
+  useEffect(
+    function loadApiData() {
+      const serverDataPromises = [axios.get(organizationsUrl)]
+      if (!isNewLandscape && landscapeId) {
+        serverDataPromises.push(axios.get(landscapeUrl))
+      }
+
+      Promise.all(serverDataPromises)
+        .then(([{ data: organizationsData }, landscapeResponse]) => {
+          if (!isNewLandscape && landscapeId && landscapeResponse) {
+            const { landscape_name, organizations: selectedOrganizations } = landscapeResponse.data
+            const landscapeDataFormattedForForm = {
+              landscape_name,
+              selectedOrganizations
+            }
+            resetForm(landscapeDataFormattedForForm)
+          }
+
+          setOrganizationOptions(organizationsData)
+          setIsLoading(false)
+        })
+        .catch((error) => {
+          setIsLoading(false)
+          if (error?.response?.status === 404) {
+            setDoesLandscapeExist(false)
+          } else {
+            toast.error(language.error.apiLoad)
+          }
+        })
+    },
+    [landscapeUrl, resetForm, isNewLandscape, landscapeId, organizationsUrl]
+  )
+
+  const formatDataForSubmit = ({ landscape_name, selectedOrganizations }) => {
+    const organizationIds = selectedOrganizations.map((organizationObject) => organizationObject.id)
+
+    return { landscape_name, organizations: organizationIds }
+  }
+
+  const postNewLandscape = (formData) => {
+    axios
+      .post(landscapesUrl, formatDataForSubmit(formData))
+      .then(({ data: { landscape_name } }) => {
+        setIsSubmitting(false)
+        toast.success(language.success.getCreateThingSuccessMessage(landscape_name))
+        navigate('/landscapes')
+      })
+      .catch((error) => {
+        setIsSubmitting(false)
+        setIsSubmitError(true)
+        toast.error(
+          <SubmitErrorWithExtraErrorContent extraErrorContent={error.response.data.error} />
```

Comment:
Could have just done a copy paste, instead of making this component which is weird. Also dont love using server for user language. "Balancing tradeoffs"

Reply:
See PR description for more on that